docs/blog/2023-09-15-zmk-sotf-6.md

```diff
@@ -0,0 +1,285 @@
+- Aurora Helix [#1873](https://github.com/zmkfirmware/zmk/pull/1873) - [petejohanson]
+- ZMK Uno shield [#1576](https://github.com/zmkfirmware/zmk/pull/1576) - [petejohanson]
+- Waterfowl [#1554](https://github.com/zmkfirmware/zmk/pull/1554) - [JW2586](https://github.com/JW2586)
+- Kyria Rev 3 [#1627](https://github.com/zmkfirmware/zmk/pull/1627) - [petejohanson]
+- Leeloo v2 and Leeloo-Micro [#1762](https://github.com/zmkfirmware/zmk/pull/1762) - [ClicketySplit](https://github.com/ClicketySplit)
+- Spaceman Pancake [#1400](https://github.com/zmkfirmware/zmk/pull/1400) - [jasonhazel](https://github.com/jasonhazel)
+- Reviung5 [#1548](https://github.com/zmkfirmware/zmk/pull/1548) - [zblesk](https://github.com/zblesk)
+
+## New Boards
+
+- RP2040 boards, including Sparkfun Pro Micro, Adafruit KB2040 and Seeeduino Xiao RP2040 were added as part of the Zephyr 3.2 upgrade in [#1499](https://github.com/zmkfirmware/zmk/pull/1499) - [petejohanson]
+- Puchi BLE [#1445](https://github.com/zmkfirmware/zmk/pull/1445) - [BenRoe](https://github.com/BenRoe)
+- nRFMicro 1.3/1.4 (nRF52833) [#912](https://github.com/zmkfirmware/zmk/pull/912) - [pashutk](https://github.com/pashutk)
+- nRF5340 DK [#1562](https://github.com/zmkfirmware/zmk/pull/1562) - [joelspadin]
+- PillBug [#1530](https://github.com/zmkfirmware/zmk/pull/1530) - [kylemccreery](https://github.com/kylemccreery)
+- Preonic Rev 3 [#1575](https://github.com/zmkfirmware/zmk/pull/1575) - [jeromeOlivier](https://github.com/jeromeOlivier)
+- Corne-ish Zen v2 [#1498](https://github.com/zmkfirmware/zmk/pull/1498) and v1 [#1593](https://github.com/zmkfirmware/zmk/pull/1593) - [LOWPROKB] and [caksoylar]
+- Polarity Works CKP family of boards [#1547](https://github.com/zmkfirmware/zmk/pull/1547) - [ReFil](https://github.com/ReFil)
+
+## Coming Soon!
+
+<!-- TODO -->
+
+Some items listed in the last coming soon section are still under active development.
+
+- Caps/Scroll/Num Lock LED support
+- Mouse Keys
+- Wired split support
+- More modular approach to external boards/shields, custom code, user keymaps, etc.
+- More shields and boards
+
+## Statistics
+
+Some statistics of interest for ZMK:
+
+- GitHub (lifetime stats)
+  - 163 Contributors
+  - 1239 Closed PRs
+  - 1845 Stars
+  - 1918 Forks
+- Discord Chat
+  - 7865 total registered (130% up from last SOTF!)
+- Website (last 30 days) <!-- TODO -->
+  - 35.9K page views
+  - 3.29K new users
+
+## Sponsorship
+
+[Open Collective](https://opencollective.com/zmkfirmware)
+[Pete Johanson](https://github.com/sponsors/petejohanson)
+(detail what is used for what)
+
+## Thanks!
+
+<!-- TODO -->
+
+As we approach the two year birthday for ZMK, I am reminded of how far we have come in such a short time, in large part thanks to the _amazing_ community that has grown around it. I am so grateful to have so many contributors, testers, and user believing in the project and helping make it a joy to work on.
```

Comment:
I actually think it'd be great to hear what *you're* thankful for! It's your blog post.

Reply:
Alright, I will write down some thoughts 😄 Let me know if you'd like to add anything additional.